lib/src/protos/mod.rs

```diff
@@ -1,15 +1,9 @@
-// Copyright 2020 The Jujutsu Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-include!(concat!(env!("OUT_DIR"), "/protos/mod.rs"));
+pub mod op_store {
```

Comment:
Copyright header? This file is not generated, right?

Reply:
This is now generated; see the `include_file` call in `gen-protos`. We could leave it handwritten, but it'd be one more thing to manually update whenever we add a proto file.